### PR TITLE
Add rubygem-smart_proxy_dhcp_infoblox package

### PR DESCRIPTION
**Pull request by mbacovsky**
For plugin updates, please indicate which repos this should be built into:

* [x] Nightly
* [x] 1.13
* [ ] 1.12
* [ ] 1.11

See Foreman's [plugin maintainer documentation](http://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Plugin#Release-strategies) for more information.

---
Scratch builds are:
EL7: https://koji.katello.org/koji/taskinfo?taskID=527742
F24: https://koji.katello.org/koji/taskinfo?taskID=527743

The RPM is installable on nightly.

### CODE REVIEW

File: rubygem-smart_proxy_dhcp_infoblox/rubygem-smart_proxy_dhcp_infoblox.spec [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+Source0: http://rubygems.org/gems/%{gem_name}-%{version}.gem
+Requires: foreman-proxy >= 1.13.0
+%if 0%{?rhel} == 6
+Requires: ruby(abi)
+%else
+Requires: ruby(release)
+%endif
+Requires: ruby
+Requires: ruby(rubygems)
+Requires: rubygem(infoblox)
```
Comment (domcleal) [inline rubygem-smart_proxy_dhcp_infoblox/rubygem-smart_proxy_dhcp_infoblox.spec]: Same comment as dns_infoblox, this dependency should also be added to the gem itself, it's not expressed there.

**Comment by iNecas** [comment]
I've tested the package on el7 and everything works fine with it.

**Comment by domcleal** [comment]
Merged and building, thanks @mbacovsky.

**Comment by iNecas** [comment]
Can we get this pulled into 1.13?

**Comment by domcleal** [comment]
Seems it already was in rpm/1.13 (5d727e3) but I hadn't built it. Apologies, now building - thanks for spotting it.

**Pull request merged**